packages/react-scripts/bin/react-scripts.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 var spawn = require('cross-spawn');
+var getDebugFlag = require('../utils/getDebugFlag');
```

Comment:
For this to work after the eject, you'll probably have to add the right path [here](https://github.com/facebookincubator/create-react-app/blob/21b00444120c82f1a1398d4ad515459856d24732/packages/react-scripts/scripts/eject.js#L52).